### PR TITLE
merge request

### PR DESCRIPTION
**Pull request by marlonosia**
please check and merge my request

### CODE REVIEW

File: screens/Register/index.jsx [hunk 1]
```diff
@@ -1,10 +1,11 @@
- import { View, Text, TouchableOpacity,Image,TextInput,setRememberMe,rememberMe,navigateToRegister,Button} from "react-native";
+ import { View, Text, TouchableOpacity,Image,TextInput,setRememberMe,rememberMe,navigateToRegister,Button,value} from "react-native";
  import React from "react";
  import { SafeAreaView } from "react-native-safe-area-context";
```
Comment (mgacrama211176) [inline screens/Register/index.jsx:3]: Meron na to sa react-native. 

You just need to call the element SafeAreaView.
https://reactnative.dev/docs/safeareaview

File: screens/Register/index.jsx [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
  import logo from "../../assets/logo.png";
 
 
 
+
  const Register = ({ navigation,phone }) => {
```
Comment (mgacrama211176) [inline screens/Register/index.jsx:9]: San to galing si phone?

This is not needed.

File: screens/Register/index.jsx [hunk 3]
```diff
@@ -106,6 +107,7 @@
               }}>
                 <TextInput
                 placeholder=" Enter phone number"
+                value={value}
```
Comment (mgacrama211176) [inline screens/Register/index.jsx:110]: You are placing value to a non-existent value. 

Either you manually put a default value or use a UseState method to store your value.

https://www.w3schools.com/react/react_usestate.asp

File: screens/Register/index.jsx [hunk 4]
```diff
@@ -23,7 +24,7 @@
                 }}>
                   <Image source={{
                 width: "100%",
-                height: "100%",
+                height: "100%", 
                 resizeMode: "contain",
               }} />
```
Comment (mgacrama211176) [inline screens/Register/index.jsx:29]: Incorrect usage of the image element:

https://reactnative.dev/docs/image

File: screens/Register/index.jsx
Comment (mgacrama211176) [inline screens/Register/index.jsx:1]: For the textInput there is a way for the default input data to always stay on lowerCase or upperCase:

https://reactnative.dev/docs/text-style-props#texttransform

Comment (mgacrama211176) [inline screens/Register/index.jsx:1]: For email text Input Check docs for auto checking if the inputed data is an email:

https://reactnative.dev/docs/textinput

Comment (mgacrama211176) [inline screens/Register/index.jsx:1]: Skills Offered should be Select Element not TextInput

**Review by mgacrama211176 — CHANGES_REQUESTED** [review]
Good Job but refer on docs for correct structure building.

File: screens/Register/index.jsx (hunk 2 above)
Comment (marlonosia) [reply screens/Register/index.jsx:9]: okay sir I remove

File: screens/Register/index.jsx [hunk 5]
```diff
@@ -1,26 +1,272 @@
-import { View, Text, TouchableOpacity } from "react-native";
+import {
+  View,
+  Text,
+  onPress,
```
Comment (mgacrama211176) [inline screens/Register/index.jsx:4]: remove unused imports

File: screens/Register/index.jsx [hunk 6]
```diff
@@ -1,26 +1,272 @@
...
 
 const Register = ({ navigation }) => {
   //Navigate to next screen
   const navigateToNextScreen = () => {
     //navigate to face verify screen
     navigation.navigate("FaceVerify");
+    const options = [
+      { label: "Amo",  },
+      { label: "Empleyado",}
+    ];
```
Comment (mgacrama211176) [inline screens/Register/index.jsx:30]: options can be directly called on line#53

File: screens/Register/index.jsx [hunk 7]
```diff
@@ -1,26 +1,272 @@
...
+        ]}
+        testID="options"
+        initial={0}
+        buttonColor={"#437456"}
+        borderRadius={10}
+        boarderColor={"#06C09F"}
+        backgroundColor={"white"}
+        
+       
+
```
Comment (mgacrama211176) [inline screens/Register/index.jsx:65]: Remove some of the excess spacings.

**Review by mgacrama211176 — CHANGES_REQUESTED** [review]
Almost there. Good Job. Errors trappings na next.

File: screens/Register/index.jsx (hunk 7 above)
Comment (marlonosia) [reply screens/Register/index.jsx:65]: okay

File: screens/Register/index.jsx [hunk 8]
```diff
@@ -1,27 +1,282 @@
...
 
 const Register = ({ navigation }) => {
   //Navigate to next screen
   const navigateToNextScreen = () => {
     //navigate to face verify screen
     navigation.navigate("FaceVerify");
   };
+
+  const [password, setPassword] = useState(""); //checker if the password and verify password is correct
+  const [verifyPassword, setVerifyPassword] = useState(""); //checker if the password and verify password is correct
```
Comment (mgacrama211176) [inline screens/Register/index.jsx:14]: Remove unsused variables.

File: screens/Register/index.jsx [hunk 9]
```diff
@@ -1,27 +1,282 @@
...
+          </View>
+          <View
+            style={{
+              borderWidth: 1,
+              width: "90%",
+              borderRadius: 10,
+              alignItems: "left",
+            }}
+          >
+            <TextInput
```
Comment (mgacrama211176) [inline screens/Register/index.jsx:116]: OnchangeText change missing. After Input what will happen to the inputted data if not stored.

File: screens/Register/index.jsx [hunk 10]
```diff
@@ -1,27 +1,282 @@
...
+              placeholder="Email Address"
+              style={{ padding: 10 }}
+              // always set Email to smallcase
+              autoCapitalize="none"
+            />
```
Comment (mgacrama211176) [inline screens/Register/index.jsx:151]: Need to make sure that the TextInput only accepts email. Needs validation.

File: screens/Register/index.jsx [hunk 11]
```diff
@@ -1,27 +1,282 @@
...
+              borderRadius: 10,
+              alignItems: "left",
+            }}
+          >
+            <TextInput
+              placeholder="Date of Birth"
+              style={{ padding: 10 }}
+              // always set Date type
+              autoCapitalize="none"
+            />
```
Comment (mgacrama211176) [inline screens/Register/index.jsx:184]: This needs to be a datepicker not a textInput

File: screens/Register/index.jsx [hunk 12]
```diff
@@ -1,27 +1,282 @@
...
+  const passwordChecker = () => {
+    //checks the passwod if they are matched and return error message
+    if (newUserData.password !== newUserData.verifyPassword) {
+      return setErrorMessage("Password did not match");
+    } else {
+      //exercise:
+      //if verifyPassword is empty, return no error message
+      return setErrorMessage("");
+    }
+  };
```
Comment (mgacrama211176) [inline screens/Register/index.jsx:45]: Add an error trapping where if the verifypassword is empty there should be no error message.

**Review by mgacrama211176 — CHANGES_REQUESTED** [review]
Almost Done. Just make sure to apply all the onChangeText to make sure that we have stored the data.

**Review by mgacrama211176 — CHANGES_REQUESTED** [review]
Almost Done. Just make sure to apply all the onChangeText to make sure that we have stored the data.

**Review by stolenfallen1 — APPROVED** [review]
If @mgacrama211176 request are done, can merge :)

**Review by mgacrama211176 — APPROVED** [review]
Good Job on doing the tasks there are still a couple of issues but the main functionality for registration is working.

Need to work on:

1. Password verification.
2. Date of Birth bug when pressed.
3. Email Verification.

This can now be merged but please check issues for immediate fix.

**Pull request merged**